README.md

```diff
@@ -80,7 +77,7 @@ At Qiskit, we feel that it is important to recognize the contributions that were
 
 ## When you find an issue in the original documentation
 
-During the process of translating Qiskit documentation, you may identify errors (mostly typos) in the original documentation. Please feel free to suggest corrections by opening an [issue](https://github.com/Qiskit/qiskit/issues/new/choose) and choose **Documentation**.
+During the process of translating Qiskit Ecosystem documentation, you may identify errors (mostly typos) in the original documentation. Please feel free to suggest corrections by opening an [issue](https://github.com/Qiskit/qiskit/issues/new/choose) and choose **Documentation**.
```

Comment:
I didn't know what to put here. Should we list the four projects and their issue trackers?

Should we list the 4 projects in general much more in this Readme? "Qiskit Ecosystem projects" is a little vague.

Reply:
Yes, I also think that putting the links to 4 projects makes more sense.

Reply:
Great. Done!